cmd/gateway-azure.go

```diff
@@ -161,6 +136,75 @@ func azureToObjectError(err error, params ...string) error {
 	return e
 }
 
+type azureUploadID [8]byte
+
+func (id azureUploadID) String() string {
+	return fmt.Sprintf("%x", id[:])
+}
+
+func mustGetAzureUploadID() *azureUploadID {
```

Comment:
this can return a `string` and `azureUploadID` can be removed as defining a new type for this is not being useful anywhere in the code.

Reply:
Done